integrations/cdi/jta-cdi/src/main/java/io/helidon/integrations/jta/cdi/NarayanaCracResource.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.jta.cdi;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import io.helidon.common.LazyValue;
+import io.helidon.common.crac.Crac;
+
+import com.arjuna.ats.arjuna.coordinator.TxControl;
+
+class NarayanaCracResource implements Crac.Resource {
```

Comment:
This couples the JTA integration to CRaC semantics. Is that something you want to do? Should this somehow be outside of this module instead?

Reply:
This is our abstraction already, we can rename it to something more neutral like `Resumable` but on other side how mentioning CRaC hurts? We have native image bits there too

Reply:
I guess all I was thinking was that I could be writing a Jakarta EE application (not a full Helidon application) and today I don't really pull in any Helidon stuff at all. Maybe I don't want or care about CRaC. But now those artifacts will show up on my classpath. If there was already an agreement that this is OK, then that's that, I guess.

Reply:
Changed the name to "Resumable" and created standalone common module